content/applications/websites/website/configuration/translate.rst

```diff
@@ -2,111 +2,138 @@
 Translations
 ============
 
-Your website is displayed in the language that matches your visitor’s browser. If the browser’s
-language has not been installed and added to your website, the content is shown in the
-:ref:`default language <translate/default-language>`. When additional languages are installed, users
-can choose their preferred language using the :ref:`language selector <translate/language-selector>`.
+The website can be translated into multiple languages to allow visitors to view the content in their
+preferred language.
```

Comment:
```suggestion
Your website can be translated into multiple languages, allowing visitors to view its content in their
preferred language.
```
As discussed, you can use the "you" if it makes sense. In the intro paragraph, it often feels more natural to use "you can do X", so I would say "your website".